src/application/VariantForms/VariantSelectionForm.tsx

```diff
@@ -66,107 +73,104 @@ export class VariantSelectionForm extends Component<Props, State> {
                   onChange={checked => {
                     if (checked) this.setState({ variantType: 'ios_token' });
                   }}
                 />
 
                 <Radio
                   className="radioBtn"
-                  name="iOS Certificate Radio"
+                  name={'variant-type'}
                   id="ios"
                   label="iOS(Certificate)"
                   description="using Apple Push Network with certificates"
-                  isChecked={this.state.variantType === 'ios'}
                   onChange={checked => {
                     if (checked) {
                       this.setState({ variantType: 'ios' });
                     }
                   }}
                 />
-              </List>
-
-              <ApplicationListContext.Consumer>
-                {({ selectVariant }: ContextInterface): ReactNode => {
-                  const createVariant = async (
-                    variant: VariantDefinition,
-                    variantType: VariantType
-                  ) => {
-                    console.log('Creating Variant');
-                    try {
-                      const newVariant = await UpsClientFactory.getUpsClient()
-                        .variants[variantType].create(
-                          this.props.app!.pushApplicationID!
-                        ) // tslint:disable-next-line:no-any
-                        .withDefinition(variant as any)
-                        .execute();
-
-                      this.setState({ variant: newVariant });
-                      await selectVariant(newVariant);
-                      this.props.onFinished(newVariant);
-                      this.props.close();
-                    } catch (error) {
-                      context.alert(error);
-                    }
-                  };
-
-                  return (
-                    <VariantEditForm
-                      type={this.state.variantType}
-                      name={this.state.variantName}
-                      onCancel={() => {
-                        this.setState({ androidVariantForm: false });
-                        this.props.close();
-                      }}
-                      onSave={async variant => {
-                        await createVariant(
-                          variant,
-                          this.state.variantType as VariantType
-                        );
-                      }}
-                    />
-                  );
-                }}
-              </ApplicationListContext.Consumer>
+              </>
             </FormGroup>
           </Form>
+          <ApplicationListContext.Consumer>
+            {({ selectVariant }: ContextInterface): ReactNode => {
+              const createVariant = async (
+                variant: VariantDefinition,
+                variantType: VariantType
+              ) => {
+                console.log('Creating Variant');
```

Comment:
Saw you deleted logs elsewhere, looks like you missed one here.

Reply:
Good catch!